src/main/java/com/iexec/common/chain/ChainDeal.java

```diff
@@ -45,16 +46,15 @@
     BigInteger workerStake;
     BigInteger schedulerRewardRatio;
 
-    public static List<String> stringParamsToList(String params){
-        List<String> listParams = new ArrayList<>();
+    public static List<String> stringParamsToList(String params) {
+        List<String> listParams;
         try {
             LinkedHashMap tasksParamsMap = new ObjectMapper().readValue(params, LinkedHashMap.class);
             listParams = new ArrayList<String>(tasksParamsMap.values());
         } catch (IOException e) {
-            e.printStackTrace();
+            listParams = Collections.singletonList(params);//the requester want to execute one task with the whole string
```

Comment:
Maybe it would be good to have a log here.

Reply:
I think its going to happen very frequently, it allows the requester sending one task (very usual) to only send `bla` instead of `{ "0" : "bla" }`.
Do you think we still need it?